### PR TITLE
Fix bug UI when create identity

### PR DESCRIPTION
**Pull request by nqhhdev**
- Issues: 
1. #1679 
2. #1693 

- Resolved:

https://user-images.githubusercontent.com/99852347/229974473-e32061a2-adb2-4ab7-bf44-82dc4a7247c2.mp4

[device-2023-04-05-102307.webm](https://user-images.githubusercontent.com/99852347/229974576-a58c5a11-97a2-40a4-9131-395bab164b33.webm)

- Responsive for language is a long text

Simulator Screen Shot - iPhone 14 - 2023-04-05 at 11 43 10
Simulator Screen Shot - iPhone 14 - 2023-04-05 at 11 43 21
Simulator Screen Shot - iPhone 14 - 2023-04-05 at 11 43 36

### CODE REVIEW

File: lib/features/identity_creator/presentation/identity_creator_controller.dart [hunk 1]
```diff
@@ -429,22 +431,39 @@ class IdentityCreatorController extends BaseController {
     if (BuildUtils.isWeb) {
       onDismissIdentityCreator?.call();
     } else {
-      popBack();
+      popBack(result: true);
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_controller.dart:434]: why need `result=true`?

File: lib/features/identity_creator/presentation/widgets/identity_input_with_drop_list_field_builder.dart [hunk 2]
```diff
@@ -46,13 +50,14 @@ class IdentityInputWithDropListFieldBuilder {
...
-                ..setContentPadding(const EdgeInsets.symmetric(
-                    vertical: BuildUtils.isWeb ? 16 : 12,
-                    horizontal: 12))
-                ..setErrorText(_error))
+          focusNode: focusNode,
+          controller: editingController,
+          textInputAction: TextInputAction.done,
+          decoration: (IdentityInputDecorationBuilder()
+            ..setContentPadding(const EdgeInsets.symmetric(
+              vertical: BuildUtils.isWeb ? 16 : 12,
```
Comment (dab246) [inline lib/features/identity_creator/presentation/widgets/identity_input_with_drop_list_field_builder.dart:58]: add indent

File: lib/features/identity_creator/presentation/widgets/identity_input_with_drop_list_field_builder.dart [hunk 3]
```diff
@@ -46,13 +50,14 @@ class IdentityInputWithDropListFieldBuilder {
...
+              horizontal: 12))
```
Comment (dab246) [inline lib/features/identity_creator/presentation/widgets/identity_input_with_drop_list_field_builder.dart:59]: same

**Review by dab246 — COMMENTED** [review]
- Please desc `root cause` and `workaround`

File: lib/features/manage_account/presentation/profiles/identities/identities_controller.dart [hunk 4]
```diff
@@ -162,10 +163,15 @@ class IdentitiesController extends BaseController {
               }
             });
       } else {
+        _hideHtmlSignature();
         final newIdentityArguments = await push(
           AppRoutes.identityCreator,
           arguments: arguments);
 
+        if(newIdentityArguments == true) {
```
Comment (dab246) [inline lib/features/manage_account/presentation/profiles/identities/identities_controller.dart:171]:
```suggestion
        if (newIdentityArguments == true) {
```

File: lib/features/manage_account/presentation/profiles/identities/identities_controller.dart [hunk 5]
```diff
@@ -287,10 +295,15 @@ class IdentitiesController extends BaseController {
...
```
Comment (dab246) [inline lib/features/manage_account/presentation/profiles/identities/identities_controller.dart:303]:
```suggestion
        if (newIdentityArguments == true) {
```

**Review by dab246 — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**